pyincore_viz/analysis/incorehousingunitallocation.py

```diff
@@ -0,0 +1,9 @@
+# YW's way
+from pyincore_viz.analysisviz import AnalysisViz
+
+
+class IncoreHousingunitallocation:
+
+    @staticmethod
+    def vis():
```

Comment:
how about "visualize()" then "viz()"? 
And, can we use "interface" class? I am not sure that python has "interface" class though

Reply:
I was trying to differentiate this from the "visualize()"  method in the analysisViz, but yeah i can name it whatever we want

Reply:
changed to visualize